### ISSUE TITLE
Fem_05 and Fem_06 wrong

### ISSUE THREAD

**Issue by CTegtmeyer** [spawaskar-cora/cora-docs#376]
I think the names for Fem_05 and Fem_06 are wrong. Fem_05 should be *Anterior-Posterior* Sagittal Subtrochanteric... and Fem_06 should be *Medial-Lateral* Transverse Subtrochanteric... , but the planes are opposite.

**Comment by jjlynch2** [comment]
The order of Medial-Lateral (Fem_05) and Anterior-Posterior (Fem_06) is correct. This change was made according to the reorganization of measurements in the UTK DCP 2.0, which now lists Anterior-Posterior (80) after Medial-Lateral (79).

**Comment by CTegtmeyer** [comment]
Okay, let me rephrase. The order is not wrong then, but currently CoRA has it listed as Medial-Lateral (Sagittal) and Anterior-Posterior (Transverse). The planes in the parentheses should be switched I believe. At least if it is to match the help docs.

**Comment by clegarde** [comment]
Ah yes! You are correct @CTegtmeyer. The help guide is correct. I may have missed the Sagittal/Transverse in ( ) when I verified the fixed measurement names in CoRA earlier. 

Measurement names should be:
Fem_05 Medial-Lateral (Transverse) Subtrochanteric Diameter
Fem_06 Anterior-Posterior (Sagittal) Subtrochanteric Diameter

**Comment by clegarde** [comment]
@SachinPawaskarUNO this is just an issue of switching (Transverse) and (Sagittal) in the current field names. This was an issue with the field names from the beginning that I missed with our earlier updates to the field names.

**Comment by SachinPawaskarUNO** [comment]
@CTegtmeyer @jjlynch2 @clegarde This data issue has been addressed. Please verify.

**Comment by clegarde** [comment]
GTG. Thanks @SachinPawaskarUNO
